README.md

```diff
@@ -179,6 +179,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :video_camera: [C Programming for Beginners](https://youtube.com/playlist?list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR)
 - :books: [Learn C Org](https://www.learn-c.org)
 - :bulb: [C Cheatsheet](https://www.codewithharry.com/blogpost/c-cheatsheet)
+- :books: [C Tutorial](https://www.scaler.com/topics/c/)
```

Comment:
The title is longer than this as I can see when I open the page. Please set it to "C Tutorial - Learn C in Detail" and do the same for the other pages.